shell/platform/glfw/flutter_glfw.cc

```diff
@@ -149,10 +149,57 @@ struct FlutterDesktopPluginRegistrar {
 
 // State associated with the messenger used to communicate with the engine.
 struct FlutterDesktopMessenger {
```

Comment:
You may as well make this a class.

Reply:
I can't make it a class because it is forward declared a struct in the C headers, so it must remain a struct in the definition.